substrate/frame/multisig/src/lib.rs

```diff
@@ -157,7 +157,28 @@ pub mod pallet {
 		/// Weight information for extrinsics in this pallet.
 		type WeightInfo: weights::WeightInfo;
 
-		/// Provider for the block number. Normally this is the `frame_system` pallet.
+		/// Query the current block number.
```

Comment:
nice. can you copy this doc for all block number provider config types?

Reply:
Coped it to proxy, recovery and vesting now. Would have to check for the other pallets since it maybe needs to be tweaked first.